CONTRIBUTING.md

```diff
@@ -93,3 +93,36 @@ For merging, you should:
        Travis to run the tests for each change you add in the pull request.
        Because testing here will delay tests by other developers, please ensure
        that the code passes all tests on your local system first.
+
+Project Style Guidelines
+------------------------
+
+In general, pysat follows PEP8 and numpydoc guidelines.  Pytest runs the unit
+ and integration tests, flake8 checks for style, and sphinx-build performs
+documentation tests.  However, there are certain additional style elements that
+have been settled on to ensure the project maintains a consistent coding style.
+These include:
+
+* Line breaks should occur before a binary operator (ignoring flake8 W503)
+* Combine long strings using `join`
+* Preferably break long lines on open parentheses rather than using `\`
+* Use no more than 80 characters per line
+* Avoid using Instrument class key attribute names as unrelated variable names:
+  * `platform`
+  * `name`
+  * `tag`
+  * `inst_id`
```

Comment:
These aren't showing up like you want. It isn't recognizing them as a sublist. 

Sorry about the individual comments and thus multiple emails.

Reply:
No worries, I'll leave this until Monday to give you more time to look at this.  If you'd like to take a stab at fixing this formatting, feel free.  Otherwise, maybe put in a suggestion where these are just listed as part of a sentence in the above bullet?

Reply:
@rfuller81 can you sort out how to get a nested list in markdown?

Reply:
@rfuller81 github markdown specifically :-P. I implemented a work-around for now, so if you can make this a nested list later that would be great.